Integrations/GitHub/GitHub.yml

```diff
@@ -730,6 +730,66 @@ script:
     - contextPath: GitHub.Commit.Verification.Payload
       description: The commit verification payload
       type: Unknown
+  - arguments:
+    - default: false
+      description: GitHub login or email address by which to filter by commit author
+      isArray: false
+      name: author
+      required: false
+      secret: false
+    - default: false
+      description: Limit number of commits
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    deprecated: false
+    description: Get list of commits
+    execution: false
+    name: GitHub-list-commits
+    outputs:
```

Comment:
after you fix the outputs issue in the code, please make sure you populate all fields in the outputs.
If any field dose not exist in the response, please remove it from the outputs because it mislead the user when building a playbook.